### PR TITLE
Remove YouTube embedding in the chat

### LINKED ISSUE #1079: Remove YouTube embedding in the chat
It doesn't provide a ton of value, I don't see many people sharing links to YouTube anyway, and it's just one more thing to maintain.

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes  #1079

### CODE REVIEW

File: webroot/index.html [hunk 1]
```diff
@@ -63,7 +63,6 @@
     <script type="preload" src="/js/web_modules/common/core-fed3ccd8.js"></script>
     <script type="preload" src="/js/web_modules/htm.js"></script>
     <script type="preload" src="/js/web_modules/@joeattardi/emoji-button.js"></script>
-    <script type="preload" src="/js/web_modules/@justinribeiro/lite-youtube.js"></script>
```
Comment (jeyemwey) [inline webroot/index.html:66]: Can you also remove the `lite-youtube` dependency from the [`web_modules` folder](https://github.com/owncast/owncast/tree/develop/webroot/js/web_modules)? It's just removing the files from the folder.

Comment (gabek) [reply webroot/index.html:66]: If you only remove files from that directory they will automatically come back. Feel free to remove the dependency from build/javascript/package.json so it won't show back up.

**Comment by Patil2099** [comment]
I think it is ready now. 
Please review it @gabek @jeyemwey 
Let me know if this is okay.

**Comment by jeyemwey** [comment]
LGTM. @gabek would you mind merging?

**Comment by gabek** [comment]
Looks like it's still listed as a dependency in `package.json`: https://github.com/owncast/owncast/blob/b01a3ed90510eddc870ed48c609f2f21de39607a/build/javascript/package.json#L8

**Comment by Patil2099** [comment]
> Looks like it's still listed as a dependency in `package.json`:
> 
> https://github.com/owncast/owncast/blob/b01a3ed90510eddc870ed48c609f2f21de39607a/build/javascript/package.json#L8

Removed now. It was at two places didn't realize.

**Comment by gabek** [comment]
Yeah, one section says what packages to install (standard package.json stuff) and the other is what files from that package get copied into the project.  Thank you for taking care of this!

**Pull request merged**